tests/jobs/csv_export2/conftest.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+
+import pytest
+from res.enkf import ResConfig
+
+TEST_DATA_DIR = os.environ.get("ERT_TESTDATA_ROOT")
```

Comment:
I'm only letting this slide as this is considered a temporary solution. In general I would like us to stay away from dragging in not open test data.

Reply:
yes... but it is already done this way for other tests as well ... agree it could slide considered a temp solution, but there are more permanent code in here with the same stuff ( correlated_observation_scaling ) . If this is not considered ok we should probably spawn an issue on this - at least for the other tests?

Reply:
I've already made an issue for this some time back ;)